index.js

```diff
@@ -44,8 +44,12 @@ module.exports = {
 		'max-len': ['error', {
 			code: 140,
 			ignoreComments: true,
+			// SVG d properties
+			ignorePattern: 'd="([\\s\\S]*?)"',
```

Comment:
Nice. Might also be nice to figure out how to do something for jsx 'text'

Reply:
Totally!
For later :D